Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM python:3.12 as nemo-run-update
```

Comment:
I'd need to refactor the workflows a bit for this, but I think it would be worth doing so. Let me compare across projects if there some barriers I'm not seeing yet